test/unit/server_test.js

```diff
@@ -1117,5 +1117,556 @@ describe("server.js tests", function () {
+              "amount": "13152.0000000",
+              "num_accounts": 177,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            },
+            {
+              "_links": {
+                "toml": {
+                  "href": ""
+                }
+              },
+              "asset_type": "credit_alphanum12",
+              "asset_code": "00796b",
+              "asset_issuer": "GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN",
+              "paging_token": "00796b_GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN_credit_alphanum12",
+              "amount": "1030.0000000",
+              "num_accounts": 19,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            },
+            {
+              "_links": {
+                "toml": {
+                  "href": ""
+                }
+              },
+              "asset_type": "credit_alphanum12",
+              "asset_code": "00796b",
+              "asset_issuer": "GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5",
+              "paging_token": "00796b_GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5_credit_alphanum12",
+              "amount": "12086.0000000",
+              "num_accounts": 173,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            },
+            {
+              "_links": {
+                "toml": {
+                  "href": ""
+                }
+              },
+              "asset_type": "credit_alphanum12",
+              "asset_code": "00838f",
+              "asset_issuer": "GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN",
+              "paging_token": "00838f_GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN_credit_alphanum12",
+              "amount": "990.0000000",
+              "num_accounts": 19,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            },
+            {
+              "_links": {
+                "toml": {
+                  "href": ""
+                }
+              },
+              "asset_type": "credit_alphanum12",
+              "asset_code": "00838f",
+              "asset_issuer": "GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5",
+              "paging_token": "00838f_GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5_credit_alphanum12",
+              "amount": "14539.0000000",
+              "num_accounts": 201,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            }
+          ]
+        }
+      };
+
+      it("requests the correct endpoint", function (done) {
+        this.axiosMock.expects('get')
```

Comment:
If each test has one response associate with it, can we move the `let` decleration of it to inside the test function? not critical, but cleans up scopes

Reply:
fixed